lib/fetch/body.js

```diff
@@ -397,7 +398,47 @@ function bodyMixinMethods (instance) {
 
       // If mimeType’s essence is "multipart/form-data", then:
       if (/multipart\/form-data/.test(contentType)) {
-        throw new NotSupportedError('multipart/form-data not supported')
+        const headers = {}
+        for (const [key, value] of this.headers) headers[key.toLowerCase()] = value
+
+        const responseFormData = new FormData()
+
+        let busboy
+
+        try {
+          busboy = Busboy({ headers })
+        } catch (err) {
+          // Error due to headers:
+          throw Object.assign(new TypeError(), { cause: err })
+        }
+
+        busboy.on('field', (name, value) => {
+          responseFormData.append(name, value)
+        })
+        busboy.on('file', (name, value, info) => {
+          const { filename, encoding, mimeType } = info
+          const base64 = encoding.toLowerCase() === 'base64'
+          const chunks = []
+          value.on('data', (chunk) => {
+            if (base64) chunk = Buffer.from(chunk.toString(), 'base64')
```

Comment:
Please add a test to cover this line

Reply:
It is not possible to decode every chunk from base64, since the chunk may be incomplete from the base64 point of view